src/pages/settings/Profile/ProfilePage.js

```diff
@@ -98,6 +96,12 @@ class ProfilePage extends Component {
             stateToUpdate = {...stateToUpdate, logins: this.getLogins(this.props.loginList)};
         }
 
+        // Update avatar in state if changed in props
+        const newAvatar = lodashGet(this.props.currentUserPersonalDetails, 'avatar');
+        if (newAvatar !== lodashGet(prevProps.currentUserPersonalDetails, 'avatar')) {
+            stateToUpdate = {...stateToUpdate, avatar: {uri: newAvatar}};
+        }
```

Comment:
Hmm one thought here: I could just update the state in `removeAvatar` instead of checking for avatar changes here 🤔

Reply:
I decided to do my idea - remove this from `componentDidMount` because I don't want to have to check avatar changes on EVERY re-render, instead we can immediately update state to default avatar when deleting the user's avatar